squidlib/src/main/java/squidpony/squidgrid/gui/gdx/SquidPanel.java

```diff
@@ -1048,34 +1054,68 @@ public void run() {
                     }
                 }))));
     }
+
     /**
-     * Starts an wiggling animation for the object at the given location for the given duration in seconds.
-     *
-     * @param x
-     * @param y
-     * @param color
-     * @param duration
-     */
-    public void tint(int x, int y, Color color, float duration) {
+	 * Like {@link #tint(int, int, Color, float)}, but waits for {@code delay}
+	 * (in seconds) before performing it.
+	 */
+    public void tint(float delay, int x, int y, Color color, float duration) {
         final Actor a = cellToActor(x, y);
         if(a == null)
             return;
-        if(duration < 0.02f) duration = 0.02f;
+        duration = clampDuration(duration);
         animationCount++;
 
         Color ac = scc.filter(a.getColor());
         a.addAction(Actions.sequence(
+        		Actions.delay(delay),
                 Actions.color(color, duration * 0.3f),
                 Actions.color(ac, duration * 0.7f),
-                Actions.delay(duration, Actions.run(new Runnable() {
```

Comment:
Yeah I thought it should have been useful at some point, but could not find why, and both in my gams and in the AI demo it was working well without it.